angularjs-portal-home/src/main/webapp/my-app/marketplace/controllers.js

```diff
@@ -235,10 +235,11 @@ define(['angular', 'jquery', 'require'], function(angular, $, require) {
           $scope.showCategories = true;
 
           currentPage = 'market';
-          if(currentCategory !== '')
-            $scope.categoryToShow = currentCategory;
-          else
-            $scope.categoryToShow = '';
+          if (currentCategory !== '') {
+$scope.categoryToShow = currentCategory;
+} else {
+$scope.categoryToShow = '';
+}
```

Comment:
ESLint fixer went in to fix this area, [eslint-config-google currently has the indent rule disabled](https://github.com/google/eslint-config-google/blob/699a43f7408160980f445fdce236110b367c2159/index.js#L199) :disappointed: 
So the fixer did not appropriately resolve.